met-web/src/components/engagement/new/create/engagmentCreateAction.tsx

```diff
@@ -22,6 +23,13 @@ export const engagementCreateAction: ActionFunction = async ({ request }) => {
     } catch (e) {
         console.error('Error updating engagement slug', e);
     }
+    formData
+        .get('users')
+        ?.toString()
+        .split(',')
```

Comment:
Is this converting an array to a string and back again? Is this a way to copy an array or something?